packages/edit-site/src/components/editor/global-styles-provider.js

```diff
@@ -95,22 +96,50 @@ export default function GlobalStylesProvider( {
 				set( contextSettings, path, newValue );
 				setContent( JSON.stringify( newContent ) );
 			},
-			getStyleProperty: ( context, propertyName, origin = 'merged' ) => {
+			getStyleProperty: (
+				context,
+				propertyName,
+				origin = 'merged',
+				resolveVars = true
```

Comment:
Why do we need this? I'd think resolving variables is always part of the algorithm this function has to perform (same for `setStyleProperty`).

Reply:
Feedback applied 👍  resolveVars was removed.